Library/Homebrew/search.rb

```diff
@@ -53,9 +53,6 @@ def search_taps(query, silent: false)
     end
 
     def search_formulae(string_or_regex)
-      # Use stderr to avoid breaking parsed output
-      $stderr.puts Formatter.headline("Searching local taps...", color: :blue)
```

Comment:
Is this being removed just because we're already using similar headers elsewhere on stdout?

Reply:
Yes, we now have 

```
==> Formulae
…

==> Casks
…
```

and I think it is obvious that we are “searching local taps”, so I don't think it's necessary to output this information.